tap_slack/transform.py

```diff
@@ -34,8 +35,10 @@ def transform_json(stream, data, date_fields, channel_id=None):
                 timestamp = record.get(date_field, None)
                 if timestamp and isinstance(timestamp, str):
                     if stream == 'messages' or stream == "threads" and date_field == 'ts':
-                        record['thread_ts'] = timestamp
                         record[date_field] = decimal_timestamp_to_utc_timestamp(timestamp)
+                        if thread_ts is None:
```

Comment:
Is the idea that if `stream==messages` then thread_ts is none? in this case it is clearer to show that explicitly via separate if statements rather than overwrite the thread_ts. For example, is it an error if `thread_ts` is `none` for the `threads` stream? WDYT?